pkg/csi/manila/identityserver.go

```diff
@@ -57,6 +57,13 @@ func (ids *identityServer) GetPluginCapabilities(ctx context.Context, req *csi.G
 				},
 			},
 		},
+		{
+			Type: &csi.PluginCapability_VolumeExpansion_{
+				VolumeExpansion: &csi.PluginCapability_VolumeExpansion{
+					Type: csi.PluginCapability_VolumeExpansion_ONLINE,
```

Comment:
@gman0 Is OFFLINE capability not applicable for manila csi?

Reply:
I think `VolumeExpansion.OFFLINE` and `VolumeExpansion.ONLINE` pertain to what happens when the volume is in use while its being expanded, rather than if it's allowed to be online/offline.

> This call MAY be made by the CO during any time in the lifecycle of the volume after creation if plugin has `VolumeExpansion.ONLINE` capability.
https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerexpandvolume

Manila doesn't care, so I think advertising online expansion only is enough. I asked at the CSI slack channel to make sure though.

Reply:
I'll send a comment here once somebody replies.

Reply:
Got a response: external-resizer doesn't check offline/online caps anyway, and they will be even deprecated from the CSI spec. See release notes at https://github.com/container-storage-interface/spec/releases/tag/v1.5.0

Reply:
Agree that manila doesn't care if share is in use or not.

Reply:
Ok